src/dforce.f90

```diff
@@ -902,6 +970,17 @@ subroutine dforce( NGdof, position, force, LComputeDerivatives, LComputeAxis)
 
 !-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!
 
+if(Lhessianallocated .and. Igeometry.eq.1) then
```

Comment:
You write a new file (that is also not hidden), add a comment on the top of this file (similar to the one you removed in hesian.f90) in grp_output explaining this.